.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-elixir@v1
+      - uses: erlef/setup-beam@v1
```

Comment:
What does this change get us? I'm not sure this is in scope for the propose PR.

Reply:
The change is for a non-deprecated action to set up Elixir with Github Actions. It's a good change, though I'm not sure it should be part of this PR.

Reply:
You’re right, I can open a new PR with just this change if you like.

Reply:
@CostantiniMatteo can you pull this out into a separate PR and then update this one? I think both should be good to go.